### PR TITLE
Create sorting_array.c

### PR DESCRIPTION
**Pull request by Pratyushraj2121**
I have done the Bubble Sort and Insertion Sort in C language keeping in mind the array size and most importantly the time complexity.

Thank You!

### CODE REVIEW

**Review by utkarsh-shekhar — CHANGES_REQUESTED** [review]
Please create two separate files for the two programs.

**Review by Pratyushraj2121 — COMMENTED** [review]
The changes asked has been countered.

Thank You!

File: c/bubble_sort.c [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/*
+BUBBLE SORT with Time Complexity : n
+int main()
```
Comment (utkarsh-shekhar) [inline c/bubble_sort.c:3]: Please uncomment the main code for both bubble_sort.c and insertion_sort.c.

File: c/sorting_array.c [hunk 2]
```diff
@@ -0,0 +1,73 @@
+/*
```
Comment (utkarsh-shekhar) [inline c/sorting_array.c:1]: This file is not necessary anymore. Please delete it.

File: c/bubble_sort.c (hunk 1 above)
Comment (utkarsh-shekhar) [reply c/bubble_sort.c:3]: What we want to do is have code which just need to be compiled without editing any file.

**Review by utkarsh-shekhar — CHANGES_REQUESTED** [review]

**Review by Pratyushraj2121 — COMMENTED** [review]
I deleted the file.

**Review by Pratyushraj2121 — COMMENTED** [review]
I have done the uncomment part. You can check both the files and suggest any further changes.

Thank You!

**Review by utkarsh-shekhar — CHANGES_REQUESTED** [review]
Nice work!

One last thing, though. Can you please format your code so that it is easier to read? You can use this online tool for that: [https://codebeautify.org/c-formatter-beautifier](https://codebeautify.org/c-formatter-beautifier)

**Review by Pratyushraj2121 — COMMENTED** [review]
The code has been formatted as suggested.

Thank You!

**Review by utkarsh-shekhar — APPROVED** [review]
Love it! <3

**Pull request merged**